rotkehlchen/chain/evm/transactions.py

```diff
@@ -166,6 +188,7 @@ def _query_and_save_transactions_for_range(
             address: ChecksumEvmAddress,
             period: TimestampOrBlockRange,
             location_string: str | None = None,
+            update_ranges: bool = True,
```

Comment:
can you explain a bit in the docstring about why `update_ranges` is here and when it's expected to be False?